.github/workflows/release.yml

```diff
@@ -95,10 +95,16 @@ jobs:
           cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
           cmake --build build --target omega_edit
 
-      - name: Move library file
+      - name: Move library file mac
+        if: startsWith(matrix.os, 'mac')
```

Comment:
What about Linux?  What I've been doing is using a test like this:

For Linux and macOS:

```
if: runner.os != 'Windows'
```

And for Windows:

```
if: runner.os == 'Windows'
```